pkg/devspace/config/versions/latest/schema.go

```diff
@@ -817,6 +819,38 @@ type RollingUpdateConfig struct {
 	Partition      *int   `yaml:"partition,omitempty" json:"partition,omitempty"`
 }
 
+// TankaConfig defines the specific tanka options used during deployment.
+type TankaConfig struct {
+	// Path is the (relative) path of the tanka environment, usually identified by jsonnetfile.json.
+	Path string `yaml:"path,omitempty" json:"path,omitempty"`
```

Comment:
Whats the difference to environmentPath?

Reply:
So, the `path` is the root path of a tanka directory structure, e.g.: 
```
./environments/default
./lib/
./vendor/
```

The `EnvironmentPath` is a reference inside that structure, which is the entry point for generating the YAML files for the described environment.

Reply:
Partially, but with many asterisks; e.g., the command can fail when there's e.g., a testing environment defined in the external dependencies.

Setting the path aligns with the usage of the tanka CLI, which is for this case: `tk apply environments/default`

Reply:
@FabianKramm , following your suggestion, I made `path` optional, making devspace able to discover the root of the tanka project based on the existence of `jsonnetfile.json` file. This should resolve this conversation.

Also, you might see I have been quite active the last month pushing changes into this branch. I have been using a fork of Devspace based on this PR to basically move our stack based on Tanka at Ankorstore into Devspace. It is now working pretty well and this exercise allowed me to identify issues in this PR and edge cases that this PR didn't cover. Now it does, and I think it is ready to be "promoted" for GA.

Are you guys (still) interested in having Tanka deployments functionality in the Devspace upstream? If so, can we make a final push so we can deliver it? I'm at your disposal for any change you think is necessary.